node/src/remote/index.ts

```diff
@@ -454,8 +454,10 @@ export class RemoteTable<T = number[]> implements Table<T> {
     }
   }
 
-  async countRows (): Promise<number> {
```

Comment:
What is the local native API? can we make them consistent. 

Same in python / rust as well.

Reply:
Could you add one comment that rust API has already created?

Reply:
https://github.com/lancedb/lancedb/blob/main/rust/lancedb/src/remote/table.rs#L64